doc/whats_new/v0.20.rst

```diff
@@ -121,6 +121,12 @@ Metrics
 - Partial AUC is available via ``max_fpr`` parameter in
   :func:`metrics.roc_auc_score`. :issue:`3273` by
   :user:`Alexander Niederbühl <Alexander-N>`.
+- Added control over the normalizer in 
+  :func:`metrics.normalized_mutual_information_score` and
+  :func:`metrics.adjusted_mutual_information_score` via the ``average_method``
+  parameter. In a future version, the default normalizer for each will become
```

Comment:
Let's call this future version 0.22, and put this changing default value in to the API Changes Summary

Reply:
The changing default value should be explained in the API Changes Summary *for 0.22*, right?

Reply:
There should be a section on deprecations for 0.20 in the API changes Summary.

Reply:
What should the text be here? Maybe "In `metrics.normalized_mutual_information_score` and
 `metrics.adjusted_mutual_information_score`, warn that `average_method`
  will have a new default value. In version 0.22, the default normalizer for each 
  will become the *arithmetic* mean of the entropies of each clustering."

Reply:
Say "In ... added the parameter ``average_method`` ... The default averaging method will change from (whatever it is in each) to arithmetic in 0.22``. I think?